src/primitive-field/primitive-field.component.ts

```diff
@@ -103,4 +106,11 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
     return this.schema['disabled'] ? -1 : 1;
   }
 
+  setTooltipPosition() {
+    let elems = document.getElementsByClassName('last');
```

Comment:
If you are getting the elements with class `last`, why the last element would have placement `bottom` ?